tutorials/neurips_challenge_quickstart.md

```diff
@@ -115,9 +128,9 @@ The following command will download and preprocess the Dolly15k dataset for the
 
 ```python
```

Comment:
I like you persistence, but I still think that it should be ```bash.
I mean even github thinks so - look at the red highlight color 😆.

Reply:
oops, yes, thanks!